include/bluetooth/direction.h

```diff
@@ -217,4 +230,23 @@ int bt_df_conn_cte_rx_enable(struct bt_conn *conn, const struct bt_df_conn_cte_r
  */
 int bt_df_conn_cte_rx_disable(struct bt_conn *conn);
 
+/**
+ * @brief Enable Constant Tone Extension request procedure for a connection.
+ *
+ * @param conn   Connection object.
+ * @param params CTE receive and sampling parameters.
+ *
+ * @return Zero in case of success, other value in case of failure.
+ */
+int bt_df_conn_cte_req_enable(struct bt_conn *conn, const struct bt_df_conn_cte_req_params *params);
+
+/**
+ * @brief Disable Constant Tone Extension request procedure for a connection.
+ *
+ * @param conn   Connection object.
+ *
+ * @return Zero in case of success, other value in case of failure.
+ */
+int bt_df_conn_cte_req_disable(struct bt_conn *conn);
+
```

Comment:
This isn't something we've done a lot in the API, but if functions are guarded by specific KConfigs (BT_DF_CONNECTION_CTE_REQ), we could add a reference to the Kconfig (`@kconfig{CONFIG_BT_DF_CONNECTION_CTE_REQ`)

Reply:
Added info: `* The function is available if @kconfig{CONFIG_BT_DF_CONNECTION_CTE_REQ is enabled.`

Reply:
Documentation build complains about unknown command @kconfig, I've changed the lines to:
`* The function is available if Kconfig CONFIG_BT_DF_CONNECTION_CTE_REQ is enabled.`

Reply:
> 

Hmm, that's odd. We use it e.g. for `bt_enable`: 

```
/**
 * @brief Enable Bluetooth
 *
 * Enable Bluetooth. Must be the called before any calls that
 * require communication with the local Bluetooth hardware.
 *
 * When @kconfig{CONFIG_BT_SETTINGS} has been enabled and the application is not
 * managing identities of the stack itself then the application must call
 * @ref settings_load() before the stack is fully enabled.
 * See @ref bt_id_create() for more information.
 *
 * @param cb Callback to notify completion or NULL to perform the
 * enabling synchronously.
 *
 * @return Zero on success or (negative) error code otherwise.
 */
int bt_enable(bt_ready_cb_t cb);
```

So if it fails, it looks like something else is wrong.

Reply:
I've added @kconfig again, lets see the output from doc-build.

Reply:
Seems to work :)